.gitlab/ci/global.yml

```diff
@@ -262,7 +262,7 @@
           fi
         fi
         if [[ -f $ARTIFACTS_FOLDER/coverage_report/.coverage ]]; then
-          if [[ "$CI_PIPELINE_SOURCE" == "schedule" ||  -n "${NIGHTLY}" || -n "${BUCKET_UPLOAD}" || -n "${DEMISTO_SDK_NIGHTLY}" ]]; then
```

Comment:
this will cause as to not measure the coverage changes

Reply:
We are ok with this for now. We will have to make sure that every change in CommonServerPython has unit tests.